### PR TITLE
chore(tests): review tests and add utils

### PR DESCRIPTION
**Pull request by jahtalab**
Part of #620

### CODE REVIEW

File: dev-utils/jasmine.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+/**
+ * Conditional describe
+ * @param {*} description
+ * @param {*} specDefinitions
+ * @param {*} condition
+ */
+export function describeIf(description, specDefinitions, condition) {
+  let describeFn = describe
+  if (arguments.length > 2) {
```
Comment (vigneshshanmugam) [inline dev-utils/jasmine.js]: merge two if's

File: packages/rum-core/test/common/apm-server.spec.js [hunk 2]
```diff
@@ -461,51 +460,56 @@ describe('ApmServer', function() {
...
+          transaction_sample_rate: '0.5',
+          etag: 'test'
+        })
+        sessionStorage.removeItem(LOCAL_CONFIG_KEY)
       })
-      sessionStorage.removeItem(LOCAL_CONFIG_KEY)
-    })
-  }
+    },
+    !testConfig.stackVersion ||
```
Comment (vigneshshanmugam) [inline packages/rum-core/test/common/apm-server.spec.js]: any reason why we need to run tests when stackVersion is undefined?

Comment (jahtalab) [reply packages/rum-core/test/common/apm-server.spec.js]: We should run as many tests as we can, so it's better to run all tests and if we want to exclude these we can always explicitly pass the stack version to the command line (e.g. in our local testing)

File: dev-utils/jasmine.js (hunk 1 above)
Comment (jahtalab) [reply dev-utils/jasmine.js]: Let's merge this one, I'll fix it in the next PR that I'm working on.

**Pull request merged**